test/Microsoft.AspNetCore.Diagnostics.Tests/DeveloperExceptionPageMiddlewareTest.cs

```diff
@@ -27,6 +28,42 @@ namespace Microsoft.AspNetCore.Diagnostics
 {
     public class DeveloperExceptionPageMiddlewareTest
     {
+        public static TheoryData GetMethodDisplayString_ReturnsTypeNameQualifiedMethodsData
+        {
+            get
+            {
+                var thisType = typeof(DeveloperExceptionPageMiddlewareTest);
+                var intParse = ((Func<string, int>)int.Parse).GetMethodInfo();
+                var dateTimeOffsetTryParse = typeof(DateTimeOffset).GetMethods()
+                    .First(m => m.Name == nameof(DateTimeOffset.TryParse) && m.GetParameters().Length == 2);
+                var genericTypeMethod = typeof(List<Process>).GetMethod(nameof(List<Process>.Remove));
+                var genericMethod = thisType.GetMethod(nameof(GenericMethod));
+                var byRefMethod = thisType.GetMethod(nameof(ByRefMethod));
+                var asyncMethod = thisType.GetMethod(nameof(AsyncMethod));
+
+                return new TheoryData<MethodBase, string>
+                {
+                    { intParse, "int System.int.Parse(string s)" },
+                    { dateTimeOffsetTryParse, "bool System.DateTimeOffset.TryParse(string input, out System.DateTimeOffset& result)" },
+                    { genericTypeMethod, "bool System.Collections.Generic.List<Process>.Remove(Process item)" },
+                    { genericMethod, $"Void {thisType}.{nameof(GenericMethod)}<TVal>(TVal value)" },
+                    { byRefMethod, $"decimal {thisType}.{nameof(ByRefMethod)}(int a, CultureInfo b, Int64& c)" },
```

Comment:
`Int64&`? The return values use `int`, `decimal`, etc... Can we make this `ref long`?